_playwright-tests/Integration/CanUpdateSystemWithTemplate.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { test, expect, cleanupTemplates } from 'test-utils';
+import { RHSMClient } from './helpers/rhsmClient';
+import { navigateToTemplates } from '../UI/helpers/navHelpers';
+import { closePopupsIfExist, getRowByNameOrUrl } from '../UI/helpers/helpers';
+import { randomName } from '../UI/helpers/repoHelpers';
+
+const templateNamePrefix = 'integration_test_template';
+const templateName = `${templateNamePrefix}-${randomName()}`;
+const regClient = new RHSMClient('RHSMClientTest');
+let firstVersion;
+
+test.describe('Test System With Template', async () => {
+  test('Test System With Template', async ({ page, client, cleanup }) => {
```

Comment:
tiny insignificant nitpick: it would be nicer (look better in the report) if the name of the test and describe were different 💅🏼✨